### DISCUSSION TITLE
정말 감사합니다 ㅠㅠ

### DISCUSSION THREAD

**Discussion by bicebersa** [changdoc/naver-vaccine-macro#219] [General]
3일정도 썼고 오늘 패치본 다운받아서 돌렸는데
40분만에 잡아서 모더나 맞고 왔습니다.
카카오 맵으로 잡았고 네이버맵은 뭔가 막아놓은건지 성공한적이 없네요
다시한번 정말 감사드립니다 ㅠㅠ

**Comment by loojods** [comment]
네이버에서 graphql 붙여넣고, 카카오에서 left by count 붙여넣고 카카오맵띄운뒤 자동새로고침 하는거 외에 더 해야할 일이 있나요 혹시?? ㅠㅠ

**Reply by changdoc** [reply]
없습니다.

**Comment by changdoc** [comment]
네이버 맵으로 예약 까지 잘 이동 되는 상황이라 공개해뒀는데 뭔가 변경사항이 생긴건지 파악이 안되고 있네요.

**Reply by bicebersa** [reply]
카카오맵보다 로딩시간이 길어서 늦은건지 잘 모르겠지만 저는 일단 카카오로만 성공했네요
